src/Importify/Main/File.hs

```diff
@@ -78,12 +80,24 @@ importifyFileOptions options srcFile = do
         InPlace   -> writeFile srcFile modifiedSrc
         ToFile to -> writeFile to      modifiedSrc
 
+class PathHandling b t where
+  toFilePath :: Path b t -> FilePath
+  toAbs :: Path Abs Dir -> Path b t -> Path Abs t
+
+instance PathHandling Rel File where
+  toFilePath = fromRelFile
+  toAbs = (</>)
+
+instance PathHandling Abs File where
+  toFilePath = fromAbsFile
+  toAbs _ p = p 
```

Comment:
This looks like a dirty hack to me. `toFilePath` is already defined in `Path` module for `Path b t`. I think it's better to pass always `Path Abs File` to `importifyFileContent` and do `getCurrentDir` outside `importifyFileContent`